src/entity/sources/SourceCategory.ts

```diff
@@ -0,0 +1,19 @@
+import { Column, Entity, PrimaryColumn, UpdateDateColumn } from 'typeorm';
+
+@Entity()
+export class SourceCategory {
```

Comment:
Actually missed it in RFC, but we should include the SEO metadata I think :)
https://dailydotdev.slack.com/archives/C069XNYMDGV/p1722930423016999?thread_ts=1722870716.569749&cid=C069XNYMDGV

Reply:
> Category Page (Template)

The SEO per Category does not change though. It is a template where we just change the value of one word.

Reply:
Guess we can leave it for now, pretty sure sooner then later product will want full descriptions (like we do on tag/source overviews now)

Reply:
Yeah, totally agree with that.